PersistentCache/Src/Alachisoft.NCache.PersistenceStore/EFDbProvider/EFPersistenceProviderBase.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) 2019 Alachisoft
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Alachisoft.NCache.Runtime.Caching;
+using PersistenceProviders;
+using PersistenceProviders.Model;
+using PersistentNCache.Model;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+
+namespace PersistentNCache.DbProvider
+{
+    public abstract class EFPersistenceProviderBase : IPersistenceProvider
+    {        
+        protected CacheStoreDbContext _dbContext;
+        protected ISerializer _serializer;
+
+        #region Initialization of Db
+        public abstract void Init(string connectionString);
+        public virtual void Dispose() { _dbContext.Dispose(); }
+        #endregion
+
+        #region private Methods
+        private IDictionary<string, StoreItem> GetStoreItemInternal(ICollection<string> keys)
```

Comment:
follow private methods syntax rule

Reply:
private methods moved to bottom of class.